saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -255,6 +257,18 @@ def update_user_fields(cls, order):
             except User.DoesNotExist:
                 order.user = None
 
+    @staticmethod
+    def avalara_active():
```

Comment:
- We shouldn't add solutions specific to a single plugin to core code. 
- Saleor should not know what avalara is.
- The saleor's responsibility is to call the expected tax method on PluginManager.
- It is plugin responsibility to handle each specific case. We don't want to have inside the Saleor code the `if else` cases related to calls on specific plugins.

Reply:
Will move that to plugin, considered placing it there myself, just one question - what exactly should we do when case from the ticket happens?
1) raise TaxError in plugin, catch it in mutation and re-raise it
2) don't calculate taxes and just return `order_line.unit_price` in plugin

_I mean `_validate_order` would get patched to return `False` in case of no address and this fragment of plugin code remains:_
```
if _validate_order(order_line.order):
    return self._calculate_order_line_unit(order_line)
return order_line.unit_price
```